src/jit/compiler.h

```diff
@@ -7707,6 +7707,10 @@ public :
         inline bool         IsReadyToRun() { return false; }
 #endif
 
+        // true if we should use the PINVOKE_{BEGIN,END} helpers instead of generating
+        // PInvoke transitions inline (e.g. when targeting CoreRT).
+        inline bool         ShouldUsePInvokeHelpers() { return (jitFlags->corJitFlags2 & CORJIT_FLG2_USE_PINVOKE_HELPERS) != 0; }
```

Comment:
I'd prefer to remain consistent with the other accessors in the case. If we believe that it is important that these methods have braces on the next line, we should change them together.